appsec/cmake/ddtrace.cmake

```diff
@@ -0,0 +1,120 @@
+include(ExternalProject)
```

Comment:
Can we please not have this?

I would like to not maintain that separately, but would prefer if there were a solution which just wraps around the "normal" phpize & configure & make process of ddtrace.

Reply:
I understand it, but this disabled by default and not used in any of the CI jobs. I don't want to impose it a maintenance burden on the tracer side.

It is only a debug/dev aid. I keep tripping on out-of-date builds of the tracer, plus I didn't find any way to builds with full debug symbols on the C and rust side, esp. on release builds. With this I can keep builds of several php versions without recompiling ddtrace all the time. I just need several directories where I configure cmake with a different php-config.